File: File/armcheck.c
#include <stdio.h>

int solve(int t) {
	int a = 0, cnt, b;
	cnt = t;
	while(t) {
		b = t%10;
		a += (b*b*b);
		t/=10;
	}
	if(a == cnt)
		return 1;
	return 0;
}

int main() {
	int t;
	printf("Enter the number: \n");
	scanf("%d", &t);
	if(solve(t))
		printf("The number is an armstrong number\n");
	else
		printf("The number is not an armstrong number\n");
	return 0;
}

File: python2/armcheck.py
#!/usr/bin/python3

t = int(input("Enter the number: "))
num = t
a = []
while(t):
    a.append(t%10)
    t/=10
    t = int(t)

l = len(a)
n = 0
for i in a:
    n += i**l
if(n == num):
    print("The number is an armstrong number")
else:
    print("The number is not an armstrong number")